server.js

```diff
@@ -6,9 +6,12 @@ process.on('SIGTERM', () => process.exit(0))
 const Koop = require('koop')
 const koop = new Koop()
 
-// Install the Sample Provider
-const provider = require('./')
-koop.register(provider)
+// Install Craigslist Provider
+const koopCraigslist = require('./koop-provider-craigslist')
```

Comment:
let's keep the sample provider really simple. let's create a new sample one that has token auth and link to it from here.